django_projects/locallibrary/catalog/templates/base_generic.html

```diff
@@ -17,12 +17,29 @@
       {% load i18n %}
         <ul class="sidebar-nav">
           <li><a href="{% url 'index' %}">{% trans 'Home' %}</a></li>
-          <li><a href="">{% trans 'All books' %}</a></li>
+          <li><a href="{% url 'books' %}">{% trans 'All books' %}</a></li>
           <li><a href="">{% trans 'All authors' %}</a></li>
         </ul>
       {% endblock %}
       </div>
-      <div class="col-sm-10 ">{% block content %}{% endblock %}</div>
+      <div class="col-sm-10 ">
```

Comment:
Chỗ này em đang xóa bay luôn cái block content rồi, nên nó ko tìm thấy để chèn vào tương ứng
```suggestion
      <div class="col-sm-10 ">{% block content %}{% endblock %}
          {% block pagination %} .... {% endblock %}
      </div>
```

Reply:
Em fix lại rồi ạ, chị check cho em.